auslib/admin/views/scheduled_changes.py

```diff
@@ -82,7 +82,7 @@ def _post(self, sc_id, form, transaction, changed_by):
         for k, v in form._fields.iteritems():
             # sc_data_version is a "special" column, in that it's not part of the
             # primary data, and shouldn't be updatable by the user.
-            if k == "sc_data_version":
+            if k == "sc_data_version" or v.data is None:
```

Comment:
Unfortunately, this has made it impossible to update values to None in the Rules Scheduled Changes :(. Eg: if I scheduled a change, then try to update that scheduled change to change "product" to an empty value, it accepts the update but doesn't make the change. I _think_ the right thing to do here is to make sure the UI doesn't set "data" in the request if no new file is being uploaded - the if block below this should take care of handling that correctly.

Sorry for misleading you by suggesting modifying this yesterday =/.